### DISCUSSION TITLE
advanced views [May 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2018] [Discord Threads]
> 🙋‍♂️ **Guineu**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### they are yet to be released. Kanban there has been some progress. We're majorly evening out new release made last week. These will be looked into later.  
#  
> 🙋‍♂️ **Guineu**  
#### That's what I figured. Thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968840915005345882)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
